src/core.jl

```diff
@@ -1,22 +1,44 @@
 
 import StatsBase.fit
 import StatsBase.predict
+import StatsBase.span
```

Comment:
- If you need to use `using StatsBase`, you might as well remove the imports. You will also need to do (`StatsBase.fit( )`, `StatsBase.predict()` etc.) to all the methods below.
- Move the line `using StatsBase` into the `NaiveBayes.jl` file.
- Add StatsBase to the `REQUIRE` file. It's currently missing.